perf/perf_24x7_hardware_counters.py

```diff
@@ -46,12 +47,14 @@ def setUp(self):
         """
         smm = SoftwareManager()
         detected_distro = distro.detect()
-        if 'ppc' not in process.system_output("uname -p", ignore_status=True).decode("utf-8"):
+        if 'ppc' not in process.system_output("uname -a", ignore_status=True).decode("utf-8"):
```

Comment:
the earlier condition also woks well so i don't think change is requrie

Reply:
No uname -p is not portable so it doesn't work on all distro

Reply:
ah but i did not see any issue can you please list out which distro you are facing a issue

Reply:
-p sounds supported but may report value unknown - Got that on PowerNV debian system. Going to investigate why.

Reply:
reverting